### PR TITLE
feat(subprocess): display notification when clicking on "assign" button

### LINKED ISSUE #76: [FEAT] case monitoring sub-process: when clicking on the "assign button", display a notification
This is an improvement following #67.
Clicking on the button, close the popover, display a notification (as in #47), then go back to the main process view.

### PR DESCRIPTION
**Pull request by assynour**
closes #76

### CODE REVIEW

File: src/case-monitoring/sub-process.ts [hunk 1]
```diff
@@ -90,8 +95,15 @@ class SubProcessTippySupport extends AbstractTippySupport {
     }
   }
 
-  private readonly assignResource = (_event: Event) => {
+  private readonly assignResource = (event: Event) => {
+    // Return to main process
     displayView('main');
+
+    // Show notifaction
```
Comment (tbouffard) [inline src/case-monitoring/sub-process.ts:102]: **suggestion:** typo
```suggestion
    // Show notification
```

File: src/case-monitoring/sub-process.ts [hunk 2]
```diff
@@ -90,8 +95,15 @@ class SubProcessTippySupport extends AbstractTippySupport {
...
+    const button = event.currentTarget as HTMLButtonElement;
+    const row = button.closest('tr')!;
+    const toastMessage = `An assignment notification is sent to ${row.cells[0].textContent ?? ''}`;
```
Comment (tbouffard) [inline src/case-monitoring/sub-process.ts:105]: **praise:** a smart way to get the content 😸

File: src/case-monitoring/sub-process.ts [hunk 3]
```diff
@@ -90,8 +95,15 @@ class SubProcessTippySupport extends AbstractTippySupport {
...
   }
 
-  private readonly assignResource = (_event: Event) => {
+  private readonly assignResource = (event: Event) => {
+    // Return to main process
     displayView('main');
+
+    // Show notifaction
+    const button = event.currentTarget as HTMLButtonElement;
+    const row = button.closest('tr')!;
```
Comment (tbouffard) [inline src/case-monitoring/sub-process.ts:104]: **praise:** a smart way to get the content 😸

File: src/case-monitoring/sub-process.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (assynour) [reply src/case-monitoring/sub-process.ts]: fixed in [91050b7](https://github.com/process-analytics/bonita-day-demo-2023/pull/91/commits/91050b756dc234082f06f6efbb3ba00ce899e765)

**Pull request merged**